### PR TITLE
Update CHANGELOG for #2702

### PR DESCRIPTION
**Pull request by mpeddada1**

### CODE REVIEW

File: jib-gradle-plugin/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 ### Fixed
+- Fixed `NullPointerException` during preconditions validation (in Java 9+) when checking for presence of `null` value in ImmutableCollection.([#2702](https://github.com/GoogleContainerTools/jib/issues/2702))
```
Comment (chanseokoh) [inline jib-gradle-plugin/CHANGELOG.md:11]: Add a blank line after `### Fixed`.

The message sounds too technical and not so helpful to the end user. The CHANGELOG is for the end user where they want to quickly find relevant bug fixes and versions that matter to them.

```suggestion
- Fixed `NullPointerException` during input validation (in Java 9+) when configuring Jib parameters using certain immutable collections (such as `List.of()`). ([#2702](https://github.com/GoogleContainerTools/jib/issues/2702))
```

File: jib-maven-plugin/CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 ### Fixed
+- Fixed `NullPointerException` during preconditions validation (in Java 9+) when checking for presence of `null` value in ImmutableCollection.([#2702](https://github.com/GoogleContainerTools/jib/issues/2702))
```
Comment (chanseokoh) [inline jib-maven-plugin/CHANGELOG.md:11]: I think we can skip Maven, as I don't think this occurs in Maven.

File: jib-gradle-plugin/CHANGELOG.md (hunk 1 above)
Comment (mpeddada1) [reply jib-gradle-plugin/CHANGELOG.md]: This is much better! Thank you

File: jib-maven-plugin/CHANGELOG.md (hunk 2 above)
Comment (mpeddada1) [reply jib-maven-plugin/CHANGELOG.md:11]: Ah you're right, good catch.

**Pull request merged**